src/components/modules/blockManager.ts

```diff
@@ -277,6 +282,7 @@ export default class BlockManager extends Module {
     needToFocus = true,
     replace = false,
     tunes = {},
+    config,
```

Comment:
Two things looks strange:

1) `config` means the Tool's config, but we're already passing the `tool`. It could be look like a redundant property (that could be got via the `tool.config`) from some perspective. Maybe we need to clarify naming.
2) `config` is passing again and again through many methods. Maybe we could get this information at the lowest level once?

Reply:
I guess `config` either could be renamed to `configOverrides` or replaced with id of selected toolbox entry. 
Anyway I don't see how passing a param could be avoided -- we need to know which toolbox entry was clicked when inserting a new block.